sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubClientBuilder.java

```diff
@@ -280,6 +290,71 @@ public EventHubClientBuilder retry(RetryOptions retryOptions) {
         return this;
     }
 
+    /**
+     * Sets the position within the partition where the consumer should begin reading events.
+     *
+     * @param eventPosition Position within an Event Hub partition to begin consuming events.
+     * @return The updated {@link EventHubClientBuilder} object.
+     */
+    public EventHubClientBuilder startingPosition(EventPosition eventPosition) {
```

Comment:
startingPosition or initialPosition? What are other languages using?

Reply:
.net uses it. So I copied from them.